cg/meta/orders/fastq_submitter.py

```diff
@@ -61,7 +61,7 @@ def create_maf_case(self, sample_obj: models.Sample):
             data_analysis=Pipeline(Pipeline.MIP_DNA),
             data_delivery=DataDelivery(DataDelivery.ANALYSIS_FILES),
             name="_".join([sample_obj.name, "MAF"]),
-            panels=None,
+            panels=["OMIM-AUTO"],
```

Comment:
I think this is called `OMIM-AUTO`. Should we add a constant class in: cg/constants/gene_panel.py that defines the panel ids? And also make the `MASTER_LIST` use that class

Reply:
I have now changed MasterList to a constant class, and refactored it everywhere I could see it be used.